### ISSUE TITLE
Can search frameworks by name

### ISSUE THREAD

**Issue by TsubasaK111** [TsubasaK111/MFW#6]
## Epic

As a **visitor**,
I can **discover amazing strategy frameworks**
So that I can **acquire easy, powerful ideas** relevant to my life's problems.  

*See the [Epics](https://github.com/TsubasaK111/MFW/tree/master/docs/epics)
## Value Statement

As a **visitor** looking for a framework related to a specific term,
I can easily **search for any framework**
So I can **discover powerful, relevant frameworks quickly**.  
## Definition of Done:
- [ ] A visitor can follow the instructions in the 'Get Started' page for engineers to:
  - [x] start a challenge,
  - [ ] access the web editor,
  - [ ] locate the primary file to build the solution in, and
  - [ ] run the `codecheck` test once.

**Comment by TsubasaK111** [comment]
Ummmm, I'm quite sure the definition of done was copied from some other project:

```
  - [x] A visitor can follow the instructions in the 'Get Started' page for engineers to:
      - [x] start a challenge,
      - [x] access the web editor,
      - [x] locate the primary file to build the solution in, and
      - [x] run the `codecheck` test once.  
```

**Comment by TsubasaK111** [comment]
@TsubasaK111
Oops! Yeah that's a placeholder, I'll populate this with actual DoDs today.

**Comment by takayukioda** [comment]
> User Story: Can search frameworks by name

I don't think this title is needed. Cause we can see this title in Issue title, and label shows it's user story.

**Comment by takayukioda** [comment]
Other than a title, it seems good! let's try with this format!

**Comment by TsubasaK111** [comment]
Redundant title removed! 
Awesome, but I'll ask @jlibasco for more feedback.
